packages/modules/stock-location/src/models/stock-location.ts

```diff
@@ -1,80 +1,19 @@
-import {
-  DALUtils,
-  Searchable,
-  createPsqlIndexStatementHelper,
-  generateEntityId,
-} from "@medusajs/framework/utils"
-import {
-  BeforeCreate,
-  Entity,
-  Filter,
-  ManyToOne,
-  OnInit,
-  PrimaryKey,
-  Property,
-} from "@mikro-orm/core"
-
-import { StockLocationAddress } from "./stock-location-address"
-
-const StockLocationDeletedAtIndex = createPsqlIndexStatementHelper({
-  tableName: "stock_location",
-  columns: "deleted_at",
-  where: "deleted_at IS NOT NULL",
+import { model } from "@medusajs/framework/utils"
```

Comment:
question: There are some commented out things in this file – what should we do with those?

Reply:
Ohh sorry. Removed now :)